### ISSUE TITLE
Add new automation to populate the milestone field of pull requests

### ISSUE THREAD

**Issue by jmakhack** [jmakhack/hydrate-reminder#361]
## Task Context

Currently, the milestone field of each pull request has to get manually set. It would be very helpful to have some form of automation that does this to save unnecessary time and effort.

## Acceptance Criteria / Goals

The goal of this task is to have automation that performs the following:
- Automatically sets the milestone of an opened pull request to the same milestone of the first mentioned issue in the pull request description
- If no milestone is mentioned in the description, it should set the milestone field to `Ongoing`

This automation will most likely come in the form of a new GitHub action. Some research may need to be done to determine how much of the above is possible and what needs to be done. There may be an existing action that handles this or not.

## Additional Notes

Please refer to [CONTRIBUTING.md](https://github.com/jmakhack/hydrate-reminder/blob/master/CONTRIBUTING.md) for info on how to setup, build, run, and test this project. Feel free to ping @jmakhack in the comments below with any questions if needed. Also, join the [Discord](https://discord.gg/RTgxfFW9mS) to meet and engage with other contributors!

[<img src="https://img.shields.io/discord/1023562225790767175?color=%20%237289DA&label=discord&logo=discord&logoColor=%20%237289DA&style=for-the-badge" />](https://discord.gg/RTgxfFW9mS)

**Comment by rishu1601** [comment]
Hey @jmakhack , let me know if this is up for grab, I will like to work on it.

**Comment by jmakhack** [comment]
Go for it @rishu1601

**Comment by rishu1601** [comment]
Hey @jmakhack , I looked through various options/ developer APIs available, I think the best way to do this will be through a github app, If it's ok I will create a github app for the same.

**Comment by jmakhack** [comment]
Yeah sure, @rishu1601. That would be great! If that is too much trouble though, I'm also OK with just setting all pull request milestones to `Ongoing`.

**Comment by jmakhack** [comment]
@rishu1601, just checking in. Are you still planning to work on this?
